### DISCUSSION TITLE
smart PIR sensor

### DISCUSSION THREAD

**Discussion by Robhaas63** [milo526/homebridge-tuya-web#219] [Q&A]
I managed to add the sensor to homebridge via plug in ' TuyaWebPlatform' , I see the sensor in home bridge at page ' Homebridge Accessoires'. and it has the same name as it has in my app (Smart Life) on my iPhone. so I know for sure homebridge discovers the sensor.

My question is when change the sensor to On, it stays at Off and does not change when it detects movement. 

btw:
When I click on the icon in homebridge it changes to On, and stays at On until I click again and then it change to off again, so some kind of software switch.

it detects movement , because I see the alarm messages on my iPhone in the app Smart Life.
did I forget something to set ? or??

**Comment by milo526** [comment]
Sadly sensors are not supported with the API that this plugin uses.
It is odd to me that the device even shows up!
I would advice to remove this device in your configuration.

**Comment by Robhaas63** [comment]
Hi milo,
Thanks for the quick reply and btw Is there another plugin that this support as you know

Verstuurd vanaf mijn iPhone 7

Gr,
Rob

**Reply by milo526** [reply]
Your best bet would be to look at the homebridge-tuya-lan plugin but I don't know if they support it.
